packages/n4s/src/rules/isPositive/index.js

```diff
@@ -0,0 +1,8 @@
+function isPositive(value, arg1) {
```

Comment:
I actually think we can remove the `isPositive` rule, and instead, just use the `negativeForm` of `isNegative`.

Removing this file and using the other notation should also fix your tests.

Reply:
removed isPositive rules. and added negativeForm inside isNegative rule and exposed isPositive.